spec/factories/drivers.rb

```diff
@@ -0,0 +1,17 @@
+FactoryBot.define do
+  factory :driver do
+    name { FFaker::Name.name }
+    license_number {  }
+    mobile_number { FFaker::PhoneNumberSG }
```

Comment:
```suggestion
    mobile_number { FFaker::PhoneNumberSG.mobile_number }
```
otherwise its just returning the class of `PhoneNumberSG`

this should have caused tests to fail, id recommend running tests before pushing changes

Reply:
somehow the tests worked when i tried, I'll make the changes

Reply:
it might have tried to call the attribute on the class, either way its still wrong for someone else looking at this code, a class name is not a mobile number, so explicitly setting the type of data we want from shoulda is important